recipe/meta.yaml

```
@@ -19,40 +19,41 @@ build:
 
 requirements:
   host:
-    - python >=3.6.1,<4.0
+    - python >=3.7.2,<4.0
     - pip
     - poetry-core >=1.0.0
 
   run:
     - backports.zoneinfo  # https://github.com/conda-forge/django-feedstock/issues/134
     - dodgy >=0.2.1,<0.3.0
-    - mccabe >=0.6.0,<0.7.0
+    - flake8 <6.0.0
+    - gitpython >=3.1.27
+    - mccabe >=0.7.0,<0.8.0
+    - packaging
     - pep8-naming >=0.3.3,<=0.10.0
-    - pycodestyle >=2.6.0,<2.9.0
-    - pydocstyle >=2.0.0
+    - pycodestyle >=2.9.0
+    - pydocstyle >=6.3.0
     - pyflakes >=2.2.0,<3.0
-    - pylint >=2.8.3,<3.0.0
+    - pylint >=2.8.3
     - pylint-celery ==0.3
     - pylint-django >=2.5,<2.6.0
     - pylint-flask ==0.6
     - pylint-plugin-utils >=0.7,<0.8
-    - python >=3.6.2,<4.0
+    - python >=3.7.2,<4.0
     - pyyaml
-    - requirements-detector >=0.7,<0.8
     - setoptconf-tmp >=0.3.1,<0.4
     - toml >=0.10.2,<0.11.0
+    - requirements-detector >=1.0.3
 
   run_constrained:
     - bandit >=1.5.1
-    - frosted >=1.4.1
     - vulture >=1.5
     - mypy >=0.600
     - pyroma >=2.4
 
 test:
   requires:
     - bandit >=1.5.1
-    - frosted >=1.4.1
     - vulture >=1.5
     - mypy >=0.600
     - pyroma >=2.4
```

Since this is already in `run_constrained` above, maybe we can drop the constraint here?

```suggestion
    - vulture
    - mypy
    - pyroma
```